$NetBSD: patch-libdwarf_Makefile,v 1.1 2011/11/26 15:50:27 dholland Exp $

Add a minor version to libdwarf.so; otherwise a bootstrapped build does
"ln -sf libdwarf.so.3 libdwarf.so.3" and then install fails with ELOOP.

This is a bug in bootstrap-mk-files but I'm going to fix it in both
places to avoid demanding rebootstrapping.

--- libdwarf/Makefile~	2011-10-27 03:47:28.000000000 +0000
+++ libdwarf/Makefile
@@ -79,6 +79,7 @@ GENSRCS=	dwarf_pubnames.c dwarf_pubtypes
 CLEANFILES=	${GENSRCS}
 
 SHLIB_MAJOR=	3
+SHLIB_MINOR=	0
 
 WARNS?=	6
 
